docs/reference-guides/block-api/block-api-versions.md

```diff
@@ -2,6 +2,9 @@
 
 This document lists the changes made between the different API versions.
 
+## Version 3 (>= WordPress 6.3)
+- The post editor will be iframed if all registered blocks have a Block API version 3 or higher. Adding version 3 support means that the block should work inside an iframe, though the block may still be rendered outside the iframe if not all blocks support version 3.
```

Comment:
Thank you for adding this :) 

I think we should also note that the block editor gets only iframed if all blocks use apiVersion 3 and also no classic meta boxes like for example Yoast SEO are rendered…

That threw me when I ran into this updating all our blocks to get to this state

Reply:
Thanks for the addition.

```js
shouldIframe={
	( 
		( hasV3BlocksOnly || ( isGutenbergPlugin && isBlockBasedTheme ) ) 
		&& ! hasMetaBoxes 
	) 
	|| isTemplateMode 
	|| deviceType === 'Tablet' 
	|| deviceType === 'Mobile'
}
```

https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-post/src/components/visual-editor/index.js#L357C7-L364C8